core/src/main/java/com/linecorp/armeria/server/HttpHeaderPathMapping.java

```diff
@@ -146,6 +147,12 @@ public PathMappingResult apply(PathMappingContext mappingCtx) {
         return PathMappingResult.empty();
     }
 
+    // TODO(minwoox) Move this method into MediaType?
```

Comment:
If performance does not matter, you could use `contentType.is(MediaType.ANY_TYPE)`.

Reply:
I needed to check if the `contentType` is the `ANY_TYPE`.
`contentType.is(MediaType.ANY_TYPE)` returns `true` for all content type. :-)